### PR TITLE
Make sure the patch directory exists

### PR DESCRIPTION
**Pull request by jamierocks**

### CODE REVIEW

File: src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy [hunk 1]
```diff
@@ -80,7 +80,11 @@ class ApplyPatchesTask extends PatchTask {
         git.fetch('origin') >> null
         git.checkout('-B', 'master', 'origin/upstream') >> null
         git.reset('--hard') >> out
-
+
```
Comment (stephan-gh) [inline src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy]: Hover this line, the spaces are unneeded.

File: src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy [hunk 2]
```diff
@@ -80,7 +80,11 @@ class ApplyPatchesTask extends PatchTask {
         git.fetch('origin') >> null
         git.checkout('-B', 'master', 'origin/upstream') >> null
         git.reset('--hard') >> out
-
+        
+        if (patchDir.exists()) {
+            patchDir.mkdirs();
```
Comment (stephan-gh) [inline src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy:85]: Create the patchDir if it already exists...? Check if the condition is really necessary, will mkdirs() fail if the dir already exists?

File: src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy (hunk 1 above)
Comment (jamierocks) [inline src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy]: Thats the issue with GH's online editor :P

File: src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy [hunk 3]
```diff
@@ -80,7 +80,11 @@ class ApplyPatchesTask extends PatchTask {
...
+        }
+
```
Comment (stephan-gh) [inline src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy]: Spaces here as well.

File: src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy (hunk 2 above)
Comment (jamierocks) [inline src/main/groovy/net/minecrell/gitpatcher/task/patch/ApplyPatchesTask.groovy:85]: Bollocks :P mkdirs doesn't fail if it already exists.

**Pull request closed without merging**